locations/spiders/mediamarkt.py

```diff
@@ -1,11 +1,27 @@
+import json
+
+from scrapy.http import Request
 from scrapy.linkextractors import LinkExtractor
-from scrapy.spiders import CrawlSpider, Rule
+from scrapy.spiders import Rule
 
 from locations.structured_data_spider import StructuredDataSpider
+from locations.user_agents import BROWSER_DEFAULT
 
 
-class MediaMarktSpider(CrawlSpider, StructuredDataSpider):
+class MediaMarktSpider(StructuredDataSpider):
     name = "media_markt"
     item_attributes = {"brand": "MediaMarkt", "brand_wikidata": "Q2381223"}
     start_urls = ["https://www.mediamarkt.de/de/store/store-finder"]
+    user_agent = BROWSER_DEFAULT
     rules = [Rule(LinkExtractor(allow=r"https://www.mediamarkt.de/de/store/"), callback="parse_sd")]
```

Comment:
Should be deleted if this is no longer a CrawlSpider?

Reply:
Yes.